.NET/Microsoft.Recognizers.Text.Number/NumberOptions.cs

```diff
@@ -6,5 +6,6 @@ namespace Microsoft.Recognizers.Text.Number
     public enum NumberOptions
     {
         None = 0,
+        ChineseNumbersWithPercent = 1
```

Comment:
I don't think this is something we should surface at the recognizer level. This is more an implementation detail of one of the Chinese number models. 
We'll do a closer analysis of the change and update the PR review soon.

Reply:
Changes in PR https://github.com/Microsoft/Recognizers-Text/pull/379

Reply:
ACK. Please, let me know if we should solve conflicts for this PR or Close it without merging.

Reply:
Let's close this one.
I'll remove the custom model test in another PR as it only target one case, and it's tested by the unit model tests anyways. It's easier that way.